libs/shared/account/account/src/lib/account.repository.ts

```diff
@@ -61,3 +61,14 @@ export function createAccount(db: AccountDatabase, account: NewAccount) {
     return true;
   });
 }
+
+/**
+ * Fetch accounts for array of uids
+ */
+export function getAccounts(db: AccountDatabase, uids: Buffer[]) {
```

Comment:
Do we typically call these 'get' or 'fetch' operations? Might be nice to have it consistent, though we're probably consistently inconsistent at the moment. 😉 

The doc string says 'fetch'. 🤷🏻

Reply:
Will have a look around the code base a bit, and at least make sure it's consistent in doc string and function name 🤦 😅

Reply:
Here's a pretty naive <kbd>git grep</kbd> which scans all of packages/** for `function get*()` or `function fetch*()` and does some basic count stats:
```sh
git grep -Phio "function (get|fetch).*?\(" -- packages | sort | uniq -c | wc -l # 175
```

Too many results to paste inline, [but here's a gist](https://gist.github.com/pdehaan/9337748e8e1f64c887898fa2eea5c92c) (which might save you like 8 sec).

Reply:
`get` seems to win out. :)